src/main/java/com/epam/lowcost/controller/FlightController.java

```diff
@@ -30,6 +30,17 @@ public String findFlightById(@RequestParam Long id, Model model) {
         return "flights";
     }
 
+
+
+    @GetMapping(value = "/search")
+    public String findFlightByFromToDate(@RequestParam Map<String, String> params, Model model) {
+        model.addAttribute("flights", flightService.getByFromToDate
+                (params.get("departureAirport"), params.get("arrivalAirport"),
+                        LocalDateTime.parse(params.get("departureDate")+"T00:00:00"),
```

Comment:
What is it ?

Reply:
user inputs only date of flight and in our database we store data+time, so it is needed to add time to search in db

Reply:
I said about formatter

Reply:
didn't understand

Reply:
`params.get("departureDate")+"T00:00:00")`

Reply:
In the project you should use alone formatter for all dates

Reply:
I dont use any formatter there, I just add time to the date I get from page
What about formatter - I fixed it

Reply:
Ok, I can ask differently. Why u use concatenation ?

Reply:
Ok, got it. Replaced concatenation with appropriate method